src/arch/riscv/common_riscv.lds

```diff
@@ -80,6 +79,7 @@ SECTIONS
 
         /* large data such as the globals frame and global PD */
         *(.bss.aligned)
+        . = . + 8;
```

Comment:
Can you add a more verbose explanation here, why this padding is needed on your platforms. I have removed this, because it was completely unclear, why this is needed. Bringing it back is fine, but this time it must have a proper explanation, what is going on on this platform.

Reply:
I just tested it again, and its the same behavior I described with the DTB round up. The rootserver gets an "Virtual address already mapped" exception from the kernel.

Reply:
is it possible to wrap this change around the CONFIG_PLAT_ROCKETCHIP_ZCU102 ifdef? That way its only used for the platform its needed.

Reply:
In https://github.com/seL4/seL4/blob/master/CMakeLists.txt#L657-L669 the linker script run through the preprocessor, so an `#ifdef` could work.

Reply:
I've created https://github.com/seL4/seL4/issues/972 as a follow-up task to investigate this. So it's not necessarily a merge blocker to have this conditional code for now.

Reply:
Thanks, Axel!